### PR TITLE
Plugin: Enable JSX syntax

### PR DESCRIPTION
**Pull request by youknowriad**
JSX or not JSX

### CODE REVIEW

File: editor/inserter/button.js [hunk 1]
```diff
@@ -3,14 +3,18 @@
...
-			el( 'span', { className: 'dashicons dashicons-plus' } )
-		),
-		opened && el( Inserter )
+	const toggle = ( event ) => {
+		event.preventDefault();
+		onClick();
+	};
+	return (
+		<div className="inserter__button">
+			<a href="" className="inserter__button-toggle" onClick={ toggle }>
```
Comment (aduth) [inline editor/inserter/button.js]: Accessibility rules alert you to problematic node without the `href` so an empty `href` is added. I see through your ruse! 😆 

This should be a `<button>`.

File: editor/inserter/button.js [hunk 2]
```diff
@@ -3,14 +3,18 @@
...
+				<span className="dashicons dashicons-plus" />
```
Comment (aduth) [inline editor/inserter/button.js]: With an appropriate screen reader label too.

**Review by aduth — COMMENTED** [review]
Granted, my notes aren't really specific to the changes here. In fact, they serve to highlight a nice benefit of using JSX: that opting in to accompanying accessibility lint rules will help us capture more problems as we move forward.

File: editor/inserter/button.js (hunk 1 above)
Comment (youknowriad) [inline editor/inserter/button.js]: You got me :)

**Comment by youknowriad** [comment]
Accessibility issues fixed (I have a lot to learn in this area). Do you think we could merge this?

**Review by aduth — APPROVED** [review]

**Pull request merged**